### ISSUE TITLE
Use standard `/test` folder for tests' visibility

### ISSUE THREAD

**Issue by allentiak** [cerner/clara-rules#452]
Otherwise, the library cannot be considered reliable...

**Comment by EthanEChristian** [comment]
Hi @allentiak,
Thanks for logging this issue. Clara does have some tests, but we are always looking to add more coverage. When writing a DSL it can be a bit tricky to cover all of the bits.

Did you have a specific area you would like to see more testing, or do you have a specific test case in mind?

Thanks in advance

**Comment by allentiak** [comment]
Now that you mention it, I checked the repo's structure, and intdeed it has some tests. I just got curious because I could not find a top-level `src` folder...
I'll change the title accordingly.

**Comment by mrrodriguez** [comment]
@allentiak plenty of clj projects follow the traditional "maven" project tradition. Clara started off under that sort of structure. There is no compelling reason to make this change for the sake of "standards". Leiningen is quite flexible in project structure & it is setup correctly.

It isn't worth making the git diff thrash of a change like this for no other gain.

**Comment by EthanEChristian** [comment]
I guess I am in agreement with @mrrodriguez here, looking at some of the core libraries for clojure:
[core.logic](https://github.com/clojure/core.logic)
[spec-alpha2](https://github.com/clojure/spec-alpha2)
[tools.logging](https://github.com/clojure/tools.logging)
[core.async](https://github.com/clojure/core.async)

While lein defaults to constructing new projects with a test root of `test` the source of truth should always be specified in the `project.clj` under `:test-paths`.

**Comment by allentiak** [comment]
@mrrodriguez , @EthanEChristian Thanks for your civil comments. (Chances are, other people would have reacted in a much more aggressive way to such a radical suggestion...)
No worries, do as you feel is the best. I only reported what I thought (think) was (is) a potential problem (not following conventions).

Personally, I don't see the need of using old, legacy structures in a new project --but that's only my opinion -- maybe I did a fuzz out of nothing, and this isn't a problem at all :-)

In any case, I am happy to have realized (albeit a little late) that your project indeed _has_ tests :-)
Thanks again, and keep up the good work!
